backend/python/app/graphql/schema.py

```diff
@@ -80,5 +86,8 @@ def resolve_stories_available_for_translation(root, info, language, level):
     def resolve_story_translations_by_user(root, info, user_id, translator):
         return resolve_story_translations_by_user(root, info, user_id, translator)
 
+    def resolve_story_available_for_review_by_user(root, info, level, language):
```

Comment:
Parameter order matters: right now the imported `resolve_story_available_for_review_by_user` is being passed a string as level and an integer as language. the function in `story_query`.py is expecting a language and then levl

Did you test out the mutation? Open up localhost:5000/graphql and try
```
{
  storyAvailableForReviewByUser(language: "HINDI", level:2){
    storyId,
    language
  }
}
```

Reply:
Ah so I think what happened was I forgot to commit the changes I made to schema.py. It worked on my local system but it won't work with the current changes.